Add Mappers' Report: February newspost (#3233)
ppy/osu-wiki: news/2020-02-24-mappers-report-february.md

We even have something new in the works! The "Mapper Showcase" will be a section where each month we write about one mapper who deserves more attention. A mapper can be featured because of some unique mapping style, amazing storyboards or even just outstanding hitsounding. And don't worry, the game mode will switch every month. No one will be left out!

With that being said, let's begin right away with the news!

This month's issue of the Mappers' Report contains following topics:

- **[Ranking Criteria Changes](#rcchanges)**
- **[NAT Meetings](#natmeetings)**
- **[Within the Beatmap Nominators and NAT](#withinbnnat)**
- **[Mapping Contests](#mapping-contests)**
- **[Summary](#summary)**

## <a id="rcchanges"></a>Ranking Criteria Changes

A new month brings a new avalanche of updates to the Ranking Criteria (RC). These past two months were especially productive with the implementation of rules!

> ### General

Reviewer: ```suggestion
### All modes
```
more descriptive, "General" is ambiguous until you release that later sections are about modes

- **[Sped Up Ver. Markers](/wiki/Ranking_Criteria#rules.2):** Songs that have been edited to have a higher tempo are now required to use a (Sped Up Ver.) marker in the title field. For songs where the genre is Techno, Trance, Dance, or other similar genres, (Nightcore Mix) will be used instead.
- **[Hitsound File Length](/wiki/Ranking_Criteria#rules.4):** The minimum length for hitsound files has been reduced from 100ms to 25ms, after testing showed that 25ms hitsound files will behave correctly in-game.
- **[Cutting Songs Guideline Changes](/wiki/Ranking_Criteria#guidelines.4):** Cuts are required to maintain the general impression and intensity of the full song, however the guideline about the cuts having to be 1:30 or above in length has been removed.
- **[Background Requirements](/wiki/Ranking_Criteria#rules.5):** New minimum width and height sizes have been added in addition to the already present maximum width and height rules. The new minimum sizes are 160 pixels wide and 120 pixels high.

> - **[Background Quality Guideline](/wiki/Ranking_Criteria#guidelines.5):** A new guideline has been added specifying that background images must be of reasonable quality.

Reviewer: "guideline" and "must" don't usually go together

### osu!taiko

- **Drum Sampleset**: A rule was removed which forbade the use of the drum sampleset. Previously, the game had no drum sample set support for osu!taiko, but this was fixed and samples added. Thus, the rule has been removed as the sampleset is now usable.

### osu!catch

- **[HP Difficulty Guidelines](/wiki/Ranking_Criteria/osu!catch#difficulty-specific):** The difficulty specific guidelines for HP settings have been adjusted for Cup to Overdose to be in line with community standards.

> ### osu!mania

Reviewer 2: be sure to add https://osu.ppy.sh/community/forums/topics/1025805?start=unread to this section now o/

### BN Retirements

Next, we have those who have taken their leave from the group. We'd like to thank them for their tremendous work and effort within the BNs and wish them the best with their newly obtained freedom!

- **osu!:** [Trynna](https://osu.ppy.sh/users/2652951)
- **osu!taiko:** [qoot8123](https://osu.ppy.sh/users/766371)
- **osu!catch:** [Sanyi](https://osu.ppy.sh/users/7496029), [Du5t](https://osu.ppy.sh/users/6053071) & [Rocma](https://osu.ppy.sh/users/566276)
- **osu!mania:** [Tailsdk](https://osu.ppy.sh/users/6751666), [Critical_Star](https://osu.ppy.sh/users/3793196), [Kyousukee](https://osu.ppy.sh/users/8842107) & [MEGAtive](https://osu.ppy.sh/users/3094101)

### NAT Additions

In addition to the recently added BNs, some new NAT members have been brought on!

The first addition we got on the 6h January was **[yaspo](https://osu.ppy.sh/users/4945926)** who served as a Beatmap Nominator for roughly over a year, he will now support the NAT for the osu! game mode. He will mainly help with BN evaluations and applications. He has also shown his skills in leading the NAT meetings already! Welcome aboard, yaspo!

> Our second addition we got on the 26th January is **[Greaper](https://osu.ppy.sh/users/2369776)**! With being previously a Beatmap Nominator for almost 4 months, he will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications and make sure, the Beatmap Nominator group will grow properly for osu!catch! So welcome aboard Greaper!

Reviewer: ```suggestion
Our second addition on the 26th January was **[Greaper](https://osu.ppy.sh/users/2369776)**! Previously having been a Beatmap Nominator for almost 4 months, he will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications, and make sure the Beatmap Nominator group will grow properly for osu!catch! So welcome aboard Greaper!
```

## <a id="mapping-contests"></a>Mapping Contests

> If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it through [this form](https://docs.google.com/forms/d/e/1FAIpQLSeVPi2KkcWiEsJzsEs1ir79HrP2jE_Sy2DCX_4nR_EECFTX4Q/viewform?usp=sf_link) for a chance of being featured on next month's installment of the Mappers' Report. We are looking to highlight more mapping contests from the community!

Reviewer: can't you just look at the forums when you write the post? there aren't too many mapping contests anyway. someone hosting a new contest probably won't go back submit it to this form

Reviewer 2: maybe we could also double check, but currently mapping contest threads get spread throughout language specific forums, mode specific forums, and beatmap projects forum, so constantly checking in all of these locations (and potentially others if contests are held elsewhere) is not feasible to always catch everything. so a form would be a good back up in this case if the contest organizer remembers it.

Reviewer 3: they're not likely to remember tbh, you're better off actively looking for these things

Author: Hmm.. couldn't we do both? I mean sure we can keep an eye on the forums but as Noffy said it's kinda difficulty to find all of them because sometimes they just get posted in language specific forums, sometimes in the tournament forum.

Just an idea: Could we make a thread in the Tournament Forum with the form which we also use in the news post? So the people who want to have a little advertising of their contest don't have to wait/look for a news post and the form would be available all the time.

Reviewer 3: that sounds like a good compromise, yes

Author: Alright, i created a thread. I will move it to the tournament forums (and pin it there) once the news post goes live!

Reviewer 3: not going to reference the thread in the post?

Author: Replaced the direct form link with the forum thread. Don't think we need it twice linked in one sentence.

### Monthly Beatmapping Contest: February 2020

> This month's installment of the Monthly Beatmapping Contest (MBC) has arrived! Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. If it's too late for you to make an entry given the deadline is in less than a week worry not, at the start of March a new contest will begin so you'll have your chance to participate. Keep your eyes peeled for the newspost which will be made when the March contest is ready.

Reviewer: ```suggestion
This month's installment of the Monthly Beatmapping Contest (MBC) has arrived! Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. If it's too late for you to make an entry given the deadline is in less than a week, worry not, at the start of March a new contest will begin so you'll have your chance to participate. Keep your eyes peeled for the news post that will be made when the March contest is ready.
```

Reviewer 3: technically you're referring to the newspost here and not the contest listing, if we're being nitpicky

Reviewer: also just noticed the deadline has passed, should be reworded a bit

> ### 6th osu!Mapping Olympiad

Reviewer: ```suggestion
### 6th osu! Mapping Olympiad
```

> For the avid fruit catchers of the world, the 6th osu!Mapping Olympiad has shown itself, a great opportunity to flex your fruit placing muscles! This osu!catch contest requires you to map 2 difficulties for MYLK's Waffle Wave track. Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-15-osu-mapping-olympiad-6-catch) for all the details! As this is an olympiad the winner of the tournament will be given the title of Elite Mapper. Good luck to all of those participating, you have until March 13th to submit your fruity feasts.

Reviewer: ```suggestion
For the avid fruit catchers of the world, the 6th osu! Mapping Olympiad has shown itself, a great opportunity to flex your fruit placing muscles! This osu!catch contest requires you to map 2 difficulties for MYLK's *Waffle Wave* track. Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-15-osu-mapping-olympiad-6-catch) for all the details! As this is an Olympiad, the winner of the tournament will be given the title of **Elite Mapper**. Good luck to all of those participating, you have until March 13th to submit your fruity feasts.
```

### Newspaper Cup

> The fantastic Chinese community organized tournament **Newspaper Cup** has made its yearly appearance and has now begun! This will be the contest's 6th yearly installment, making it one of the longest standing community organized mapping contests. All of the relevant information can be found [in this forum post](https://osu.ppy.sh/community/forums/topics/1023671) in both Chinese and English. You have until **March 15th** to submit your entries for one or more of the 4 available songs. Good luck to those planning on entering their creations into the contest!

Reviewer: if you bold the contest title and entry deadline here, you should do it on the other contests. or remove from here

> ## <a id="summary"></a>Summary

Reviewer: this isn't a summary 🤔

maybe something like this and rename navigation at the beginning. other news posts tend to use --- to separate the end
```suggestion
<a id="conclusion"></a>
---
```

Reviewer: header or separator, not both. see my suggestion^

Author: Okay now i understood. Fixed it!

> Make sure to tune in to the next NAT Meeting on March 1st in the [osu!dev Discord Server](https://discord.gg/ppy) and to do your best on all those fantastic mapping contests open for your submissions! Thank you for your time reading this quite hefty article. As this is our first installment of the Mappers' Report, any feedback is appreciated as we are still working hard at setting this up. Additionally, the next installments should be a bit smaller, as we had a lot to cover this time. We hope the information will prove itself useful to you!

Reviewer: ```suggestion
Make sure to tune in to the next NAT Meeting on March 1st in the [osu!dev Discord Server](https://discord.gg/ppy) and to do your best on all those fantastic mapping contests open for your submissions! Thank you for your time reading this quite hefty article. As this is our first installment of the Mappers' Report, any feedback is appreciated, as we are still working hard on setting this up. The next installments should be a bit smaller since we had a lot to cover this time. We hope the information will prove itself useful to you!
```

- **1 Year Badge:** [-Keitaro](https://osu.ppy.sh/users/3378391), [-Luminate](https://osu.ppy.sh/users/4778689), [_Kobii](https://osu.ppy.sh/users/6209713), [Bibbity Bill](https://osu.ppy.sh/users/4446810), [Cris-](https://osu.ppy.sh/users/6175280), [Crissa](https://osu.ppy.sh/users/5405836), [eiri-](https://osu.ppy.sh/users/3388410), [Nikakis](https://osu.ppy.sh/users/4351739), [Petal](https://osu.ppy.sh/users/7354729), [Scotty](https://osu.ppy.sh/users/11085809), [Sparhten](https://osu.ppy.sh/users/7601720), [spoes](https://osu.ppy.sh/users/9505704), [Tyistiana](https://osu.ppy.sh/users/1421452), [Venix](https://osu.ppy.sh/users/5999631), [Yugu](https://osu.ppy.sh/users/3161834) & [Zelq](https://osu.ppy.sh/users/8953955)

- **2 Year Badge:** [ailv](https://osu.ppy.sh/users/6933054), [Myxo](https://osu.ppy.sh/users/2202645), [Realazy](https://osu.ppy.sh/users/918297), [TheKingHenry](https://osu.ppy.sh/users/5128277) & [timemon](https://osu.ppy.sh/users/2072005)

- **3 Year Badge:** [Xinely](https://osu.ppy.sh/users/1521445)

### NAT Additions

In addition to the recently added BNs, some new NAT members have been brought on!

> The first addition we got on the 6h January was **[yaspo](https://osu.ppy.sh/users/4945926)** who served as a Beatmap Nominator for roughly over a year, he will now support the NAT for the osu! game mode. He will mainly help with BN evaluations and applications. He has also shown his skills in leading the NAT meetings already! Welcome aboard, yaspo!

Reviewer 4: ```suggestion
The first addition we got on the 6th January was **[yaspo](https://osu.ppy.sh/users/4945926)** who served as a Beatmap Nominator for roughly over a year, he will now support the NAT for the osu! game mode. He will mainly help with BN evaluations and applications. He has also shown his skills in leading the NAT meetings already! Welcome aboard, yaspo!
```

> Our second addition on the 26th January was **[Greaper](https://osu.ppy.sh/users/2369776)**! Previously having been a Beatmap Nominator for almost 4 months, he will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications, and make sure the Beatmap Nominator group will grow properly for osu!catch! So welcome aboard Greaper!

Reviewer 4: ```suggestion
Our second addition on the 26th January was **[Greaper](https://osu.ppy.sh/users/2369776)** who's been a Beatmap Nominator for almost 4 months. He will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications, and make sure osu!catch Beatmap Nominators grow properly! Welcome aboard, Greaper!
```

## <a id="mapping-contests"></a>Mapping Contests

If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it through the form in [this forum thread](https://osu.ppy.sh/community/forums/topics/1027590) for a chance of being featured on next month's installment of the Mappers' Report. We are looking to highlight more mapping contests from the community!

### Monthly Beatmapping Contest: February 2020

> This month's installment of the **Monthly Beatmapping Contest (MBC)** has arrived! Head on over to the [news post](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. If it's too late for you to make an entry given the deadline is in less than a week, worry not, at the start of March a new contest will begin so you'll have your chance to participate. Keep your eyes peeled for the news post that will be made when the March contest is ready.

Reviewer 2: ```suggestion
This month's installment of the **Monthly Beatmapping Contest (MBC)** has arrived! Head on over to the [news post](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. The entry period for February has just ended, but worry not! At the start of March the next contest will begin and you'll have your chance to participate. Keep your eyes peeled for the news post that will be made when the March contest is ready.
```

deadline for february already passed, so needs to be reworded appropriately.

- **osu!:** [Trynna](https://osu.ppy.sh/users/2652951)
- **osu!taiko:** [qoot8123](https://osu.ppy.sh/users/766371)
- **osu!catch:** [Du5t](https://osu.ppy.sh/users/6053071), [Rocma](https://osu.ppy.sh/users/566276) & [Sanyi](https://osu.ppy.sh/users/7496029)
- **osu!mania:** [Critical_Star](https://osu.ppy.sh/users/3793196), [Kyousukee](https://osu.ppy.sh/users/8842107), [MEGAtive](https://osu.ppy.sh/users/3094101) & [Tailsdk](https://osu.ppy.sh/users/6751666)

### BN Badges

> Before we move on to the NAT additions, we would like to honour our long standing members of the Beatmap Nominators! These BNs will receive, depending on the length of their tenure in the team, a shiny new badge for their profile page. We thank them all for their contribution so far and hope to see much more of them in the future! Here a list of those recently rewarded with a badge:

Reviewer 4: ```suggestion
Before we move on to the NAT additions, we would like to honour our long standing members of the Beatmap Nominators! These BNs will receive, depending on the length of their tenure in the team, a shiny new badge for their profile page. We thank them all for their contribution so far and hope to see much more of them in the future! Here is the list of those:
```
no need to repeat badge once again i guess
